### PR TITLE
Fix and expose Particle Entity emitRadiusStart

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/19496/Particle-entity-emitRadiusStart-not-exposed-and-not-working

Test plan:
- "Emit Radius Start" slider should show up in particle properties window under Emit section below "Emit Dimensions."
- Run this from the console:
```
Entities.addEntity({
	"type": "ParticleEffect",
	"position": MyAvatar.position,
	"isEmitting":true,"lifespan":"1.0","maxParticles":"1000","textures":"https://content.highfidelity.com/DomainContent/production/Particles/wispy-smoke.png","emitRate":"600","emitSpeed":"0","speedSpread":"0","emitDimensions":{"x":"1","y":"1","z":"1"},"emitRadiusStart":"0","emitOrientation":{"x":"-90","y":"0","z":"0"},"emitterShouldTrail":true,"particleRadius":"0.10000000149011612","radiusSpread":"0","radiusStart":"0.10000000149011612","radiusFinish":"0.10000000149011612","color":{"red":"255","blue":"255","green":"255"},"colorSpread":{"red":"0","blue":"0","green":"0"},"colorStart":{"red":"255","blue":"255","green":"255"},"colorFinish":{"red":"255","blue":"255","green":"255"},"emitAcceleration":{"x":"0","y":"0","z":"0"},"accelerationSpread":{"x":"0","y":"0","z":"0"},"alpha":"0.5","alphaSpread":"0","alphaStart":"0.5","alphaFinish":"0.5","particleSpin":0,"spinSpread":0,"spinStart":0,"spinFinish":0,"rotateWithEntity":false,"polarStart":0,"polarFinish":0.7853981633974483,"azimuthStart":-3.1415927410125732,"azimuthFinish":3.1415927410125732
});
```
- You should see something like this:
image
- Edit its "Emit Radius Start".  As it gets closer to 1, it should look more like this:
image

### CODE REVIEW

File: libraries/entities/src/ParticleEffectEntityItem.cpp [hunk 1]
```diff
@@ -117,26 +117,27 @@ bool Properties::valid() const {
...
+        (emission.rate == glm::clamp(emission.rate, MINIMUM_EMIT_RATE, MAXIMUM_EMIT_RATE)) &&
+        (emission.speed.target == glm::clamp(emission.speed.target, MINIMUM_EMIT_SPEED, MAXIMUM_EMIT_SPEED)) &&
+        (emission.speed.spread == glm::clamp(emission.speed.spread, MINIMUM_EMIT_SPEED, MAXIMUM_EMIT_SPEED)) &&
+        (emission.acceleration.target == glm::clamp(emission.acceleration.target, vec3(MINIMUM_EMIT_ACCELERATION), vec3(MAXIMUM_EMIT_ACCELERATION))) &&
+        (emission.acceleration.spread == glm::clamp(emission.acceleration.spread, vec3(MINIMUM_ACCELERATION_SPREAD), vec3(MAXIMUM_ACCELERATION_SPREAD)) &&
+        (emission.dimensions == glm::clamp(emission.dimensions, vec3(MINIMUM_EMIT_DIMENSION), vec3(MAXIMUM_EMIT_DIMENSION))) &&
+        (polar.start == glm::clamp(polar.start, MINIMUM_POLAR, MAXIMUM_POLAR)) &&
+        (polar.finish == glm::clamp(polar.finish, MINIMUM_POLAR, MAXIMUM_POLAR)) &&
+        (azimuth.start == glm::clamp(azimuth.start, MINIMUM_AZIMUTH, MAXIMUM_AZIMUTH)) &&
+        (azimuth.finish == glm::clamp(azimuth.finish, MINIMUM_AZIMUTH, MAXIMUM_AZIMUTH)));
```
Comment (SamGondelman) [inline libraries/entities/src/ParticleEffectEntityItem.cpp]: The above changes are just reorganization to match the order of the properties in the struct.

File: libraries/entities/src/ParticleEffectEntityItem.h [hunk 2]
```diff
@@ -177,9 +177,10 @@ namespace particle {
         Properties& operator =(const Properties& other) {
             color = other.color;
             alpha = other.alpha;
+            radiusStart = other.radiusStart;
```
Comment (SamGondelman) [inline libraries/entities/src/ParticleEffectEntityItem.h]: This fixes the property.  It wasn't being copied to the rendering side.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14314/HighFidelity-Beta-PR14262-8807b90.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14314/HighFidelity-Beta-PR14262-8807b90.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14314/HighFidelity-Beta-Interface-PR14262-8807b90.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14314/HighFidelity-Beta-PR14262-8807b90.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14314/HighFidelity-Beta-Interface-PR14262-8807b90.dmg

**Comment by SamGondelman** [comment]
wait for https://github.com/highfidelity/hifi/pull/14172

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14404/HighFidelity-Beta-PR14262-2009f58.apk

File: scripts/system/html/js/entityProperties.js [hunk 3]
```diff
@@ -671,6 +671,15 @@ const GROUPS = [
                 subLabels: [ "x", "y", "z" ],
                 propertyID: "emitDimensions",
             },
+            {
+                label: "Emit Radius Start",
+                type: "slider",
+                min: 0,
+                max: 1,
+                step: 0.1,
```
Comment (dback2) [inline scripts/system/html/js/entityProperties.js]: SliderFloat in particle explorer was using step of 0.01, should we do that here to be in line with the others?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14405/HighFidelity-Beta-PR14262-faff5f1.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14405/HighFidelity-Beta-PR14262-faff5f1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14405/HighFidelity-Beta-Interface-PR14262-faff5f1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14405/HighFidelity-Beta-PR14262-faff5f1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14405/HighFidelity-Beta-Interface-PR14262-faff5f1.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Particles look as expected

**Pull request merged**